doc/api/async_hooks.md

```diff
@@ -685,6 +685,8 @@ asyncResource.triggerAsyncId();
   object is garbage collected. This usually does not need to be set (even if
   `emitDestroy` is called manually), unless the resource's `asyncId` is
   retrieved and the sensitive API's `emitDestroy` is called with it.
+  When set to `false`, automatic `emitDestroy` behavior will only take
```

Comment:
We will probably need to rebase this part over https://github.com/nodejs/node/pull/32514/files

Reply:
Thanks for the heads up!

As that PR is now merged, I have rebased this PR and slightly changed this new sentence in the `AsyncResource` doc, so it's more consistent with #32514.